### DISCUSSION TITLE
How to make the PEG molecule with long chain polymers?

### DISCUSSION THREAD

**Discussion by Vignesh666d** [marrink-lab/polyply_1.0#386] [Q&A]
Since PEO are available in all the libraries  Martini3 and Martini3_beta but there is no PEG is available but I have read the 2013 paper and also looked the ITP files a slight structure change in the PEG OH group as per parameterization the bead is SP2 which is found in martini_v_2.0_polymers.itp but PEO generated is different from the PEG? How can we generate PEG alone? since PEG is not available in the library. Looking forward for your help @fgrunewald

**Comment by fgrunewald** [comment]
PEO and PEG are the same polymer. You can opt to add a CH2OH end-group e.g.
```
-seq OHter:1 PEO:10O Hter:1
```

The 2013 paper is long outdated. The current Martini2 PEO/PEG is from 2018. https://pubs.acs.org/doi/10.1021/acs.jpcb.8b04760

**Comment by Vignesh666d** [comment]
Thankyou @fgrunewald thankyou sir

**Comment by Vignesh666d** [comment]
Actually I am getting an error. Since I checked the martini3.itp files OH ter is mentioned and Hter is not available. I am getting an error. Whether H ter bead is there. I feel there is no bead in the parameter file.  we only need to add OH ter right sir.
polyply gen_params -lib martini3 -o PEG36.itp -name PEO -seq OHter:1 PEO:36 Hter:1
INFO - step - reading input and library files
INFO - step - reading sequence from command
INFO - step - mapping sequence to molecule
Traceback (most recent call last):
  File "/home/bio/.local/bin/polyply", line 273, in <module>
    main()
  File "/home/bio/.local/bin/polyply", line 269, in main
    subprogram(**args_dict)
  File "/home/bio/.local/lib/python3.8/site-packages/polyply/src/gen_itp.py", line 110, in gen_params
    meta_molecule = MapToMolecule(force_field).run_molecule(meta_molecule)
  File "/home/bio/.local/lib/python3.8/site-packages/polyply/src/map_to_molecule.py", line 308, in run_molecule
    _assert_blocks_in_FF(self.node_to_block.values(),
  File "/home/bio/.local/lib/python3.8/site-packages/polyply/src/map_to_molecule.py", line 87, in _assert_blocks_in_FF
    raise IOError("Couldn't find block with residue name {name}\n"
OSError: Couldn't find block with residue name Hter
in the library or input file definitions.
@fgrunewald .Looking forward to hear from you in leisure time. Thankyou

**Reply by fgrunewald** [reply]
PEO or PEG in CG representation does not require an H terminus. It is assumed to be included in the repeat unit (i.e. CH2-O-CH2 is the same as CH3-O-CH2 in CG representation). Only addition of the OH group requires an additional bead.

**Reply by Vignesh666d** [reply]
Thank you sir @fgrunewald
